management-model/src/main/java/org/terracotta/management/call/ContextualReturn.java

```diff
@@ -30,16 +34,20 @@
  *
  * @author Mathieu Carbou
  */
-public final class ContextualReturn<T> implements Serializable {
+public final class ContextualReturn<T> implements Externalizable {
 
-  private final T value;
-  private final Context context;
-  private final String capability;
+  private T value;
+  private Context context;
+  private String capability;
+
+  @Deprecated
```

Comment:
can you explain why an added method is marked Deprecated ?
I understand this constructor is there only for the sake of Externalizable that requires a no arg contructor, but still, it's confusing for developers reading this code